rest-api/test/unit_test/api_test/participant_summary_api_test.py

```diff
@@ -790,7 +782,6 @@ def testQuery_oneParticipant(self):
       'sexualOrientation': 'straight',
       'phoneNumber': '512-555-5555',
       'recontactMethod': 'email_code',
-      'language': 'en',
```

Comment:
we should still test `primary_language` somewhere for participant summary.

Reply:
Good point. I added it.